ui/raidboss/data/04-sb/raid/o3n.txt

```diff
@@ -0,0 +1,134 @@
+49.8 "Place Dark Token" sync /:Halicarnassus:22FC:/
+61.9 "The Queen's Waltz (Sword Dance)" sync /:Halicarnassus:246F:/ window 30,30
+62.2 "Cross Reaper" sync /:Soul Reaper:246B:/
+70.0 "The Playing Field" sync /:Halicarnassus:2300:/
+80.2 "The Game (symbols)" sync /:Halicarnassus:246D:/
+
+# Cave phase
+89.4 "Panel Swap" sync /:Halicarnassus:2304:/
+100.5 "The Queen's Waltz" sync /:Halicarnassus:2471:/
+110.7 "Place Token" sync /:Halicarnassus:22FB:/ window 30,30
+126.7 "Frost Breath?" # sync /:Great Dragon:2476:/
+128.8 "The Queen's Waltz (Uplift)" sync /:Halicarnassus:2471:/
+133.9 "Spellblade Blizzard III" sync /:Halicarnassus:2461:/
+134.7 "Frost Breath?" # sync /:Great Dragon:2476:/
+139.0 "Spellblade Fire III" sync /:Halicarnassus:2460:/
+142.8 "Frost Breath?" # sync /:Great Dragon:2476:/
+143.7 "Spellblade Thunder III" sync /:Halicarnassus:2462:/
+150.9 "Frost Breath?" # sync /:Great Dragon:2476:/
+151.8 "Mindjack" sync /:Halicarnassus:246[789A]:/ window 30,30
+158.9 "The Queen's Waltz (Uplift)" sync /:Halicarnassus:2471:/
+165.1 "Ribbit (avoid)" sync /:Halicarnassus:2466:/
+172.2 "Place Dark Token" sync /:Halicarnassus:22FC:/
+177.3 "Spellblade Holy" sync /:Halicarnassus:22EF:/
+184.6 "Cross Reaper" sync /:Soul Reaper:246B:/
+186.3 "Holy Edge" sync /:Halicarnassus:2463:/
+190.4 "The Queen's Waltz (Uplift)" sync /:Halicarnassus:2471:/
+202.4 "Aetherial Tear" sync /:Halicarnassus:2474:/ window 202.4,5
+
+# Intermission. Nothing of import happens here.
+216.6 "Aetherial Pull" sync /:Aetherial Tear:2475:/
+272.9 "Ultimum (Starry End)" # sync /:Halicarnassus:2477:/
+# There is unfortunately no way to continue to display 
+# Ultimum if another player escapes before the user
+
+# Post-intermission.
+300.0 "Panel Swap" sync /:Halicarnassus:2304:/ window 150.10
+305.1 "The Playing Field" sync /:Halicarnassus:2300:/
+313.3 "Ribbit (avoid)" sync /:Halicarnassus:2466:/
+320.5 "The Game (symbols)" sync /:Halicarnassus:246D:/
+329.6 "Spellblade Holy" sync /:Halicarnassus:22EF:/
+334.7 "Mindjack" sync /:Halicarnassus:246[789A]:/ window 30,30
+338.6 "Holy Blur" sync /:Halicarnassus:2464:/
+340.4 "Spellblade Thunder III" sync /:Halicarnassus:2462:/
+347.6 "Place Dark Token" sync /:Halicarnassus:22FC:/
+360.0 "Gusting Gouge" sync /:Soul Reaper:246C:/
+361.7 "Spellblade Blizzard III" sync /:Halicarnassus:2461:/
+368.9 "The Playing Field" sync /:Halicarnassus:2300:/
+377.1 "Ribbit (take)" sync /:Halicarnassus:2466:/ window 30,30
+384.2 "The Game (toad)" sync /:Halicarnassus:246D:/
+394.3 "Dimensional Wave" sync /:Halicarnassus:2465:/
+403.5 "Spellblade Holy" sync /:Halicarnassus:22EF:/
+407.6 "Spellblade Fire III" sync /:Halicarnassus:2460:/
+412.5 "Holy Blur" sync /:Halicarnassus:2464:/
+412.7 "The Queen's Waltz (Sword Dance)" sync /:Halicarnassus:246F:/ window 30,30
+421.8 "Dimensional Wave" sync /:Halicarnassus:2465:/
+
+# The blocks don't change, but if we don't track them we can't note the enrage.
```

Comment:
What does this mean?

Reply:
There's no way to differentiate the enrage cast without doing wonky stuff with junction blocks. (Her cast for the enrage is the same ID as non-enrage uses.) It may not be really necessary, since nobody sees it in the duty finder, but I enumerated the blocks as opposed to looping since there is an actual enrage.